src/components/explorer-ui.js

```diff
@@ -21,15 +49,31 @@ const ExplorerUI = () => {
         </h1>
         <LocationBar onNewUrl={setUrl} value={locationUrl} />
       </header>
-      <nav className="resourceLinks">
-        <div className="resourceLinks__location">Top Level</div>
-        <ul className="resourceLinks__nav">
-          {Object.keys(entrypointLinks).map((key, index) => (
-            <li key={`resource-link-${index}`} className="resourceLinks__link">
-              <LinkElement link={entrypointLinks[key]} />
-            </li>
-          ))}
-        </ul>
+      <nav className="menu">
+        <div className="menu__container">
+          <div className="menu__location">
+            <span className="menu__location_title">Top Level</span>
+          </div>
+          <ul className="menu__nav">
+            {Object.keys(entrypointLinks).map((key, index) => (
+              <li key={`resource-link-${index}`} className="menu__link">
+                <MenuLinkElement
+                  link={entrypointLinks[key]}
+                  next={() => setActiveMenu(1)}
+                />
+              </li>
+            ))}
+          </ul>
+        </div>
+        {loadedSchemas.map((loaded, index) => (
+          <SchemaMenu
+            key={`schema-menu-${[schema.type, ...loaded.forPath].join('.')}`}
+            forPath={loaded.forPath}
+            load={loadNext}
```

Comment:
I did it this way because in the `SchemaMenu` it's also calling `next` at the same time. 
```js
  const loadNext = name => {
    load([...forPath, name]);
    next();
  };
```
It could definitely be improved though

Reply:
the "forPath" needs to come from the `relationship.name` in the SchemaMenu - it wouldn't be available here I don't think